app/models/ads/ads-context.js

```diff
@@ -0,0 +1,30 @@
+import { inject as service } from '@ember/service';
```

Comment:
Same here. It seems that this file is strictly related to search page. What about `search-ads-context`?

Reply:
Yes, I didn't read that much into our endpoint.